packages/documentation/src/pages/platform/architecture/cms.mdx

```diff
@@ -0,0 +1,88 @@
+---
+title: VA.gov content management system overview
+tags: cms, drupal, headless, preview, graphql
+---
+
+# VA.gov content management system overview
+
+The VA is currently in the process of moving content from several other systems into one primary content management system, powered by Drupal. VA.gov takes a decoupled approach, where the cms is queried during the VA.gov build process and content is transformed into static html and deployed on the site. Drupal does not control any Veteran-facing rendering.
```

Comment:
It might be worth adding that the Drupal admin back-end is still there and used from the VA network. It seemed like that was an area of confusion before we started really getting going with the Drupal work, like a few folks thought we might be building that too.

Reply:
I mention the VA network requirement and admin urls down below, do you think that covers it?

Reply:
I think that covers the network piece but I think it would still be nice to mention the admin part here, only because non-devs were confused about that at the start of this all. Maybe something like -

```
 queried during the VA.gov build process and content is transformed into static html and deployed on the site, while editing content is done via the standard Drupal administrative back-end.
```